services/namex-pay/src/namex_pay/worker.py

```diff
@@ -266,7 +270,8 @@ async def process_payment(pay_msg: dict, flask_app: Flask):
 FLASK_APP = Flask(__name__)
 FLASK_APP.config.from_object(APP_CONFIG)
 db.init_app(FLASK_APP)
-
+queue.init_app(FLASK_APP, asyncio.new_event_loop())
```

Comment:
why is this added?
the worker is already called from a loop in run.py

Reply:
In run.py it runs QueueServiceManager. QueueService is a new queue manager added to namex-api and needs to be initialized in some way. I don't know if it's possible to change QueueService to work the same way as QueueServiceManager. What do you think?